fabric-content-registries-v0/src/main/java/net/fabricmc/fabric/mixin/content/registry/FuelRegistryMixin.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.mixin.content.registry;
+
+import com.llamalad7.mixinextras.injector.wrapoperation.Operation;
+import com.llamalad7.mixinextras.injector.wrapoperation.WrapOperation;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+
+import net.minecraft.item.FuelRegistry;
+import net.minecraft.item.Item;
+import net.minecraft.registry.tag.TagKey;
+
+import net.fabricmc.fabric.api.registry.FabricFuelRegistryBuilder;
+
+/**
+ * Implements the invokation of {@link FabricFuelRegistryBuilder} callbacks.
+ */
+@Mixin(FuelRegistry.class)
+public abstract class FuelRegistryMixin {
+	/**
+	 * Handles invoking both pre- and post-exclusion events.
+	 *
+	 * <p>Vanilla currently uses a single exclusion for non-flammable wood; if more builder calls for exclusions are added, this mixin method must be split accordingly.
+	 */
+	@WrapOperation(
+			method = "createDefault(Lnet/minecraft/registry/RegistryWrapper$WrapperLookup;Lnet/minecraft/resource/featuretoggle/FeatureSet;I)Lnet/minecraft/item/FuelRegistry;",
+			at = @At(
+					value = "INVOKE",
+					target = "Lnet/minecraft/item/FuelRegistry$Builder;removeAll(Lnet/minecraft/registry/tag/TagKey;)Lnet/minecraft/item/FuelRegistry$Builder;"
+			),
+			allow = 1
+	)
+	private static FuelRegistry.Builder build(FuelRegistry.Builder builder, TagKey<Item> tag, Operation<FuelRegistry.Builder> operation) {
+		FabricFuelRegistryBuilder.BUILD.invoker().build(builder);
```

Comment:
The `createDefault` method passes a reference int of value 200 (10 seconds), maybe it should be passed to the event too? Looking that it's public method, it could be useful for mods to create a version using different times or possibly by mojang in future for whatever reason.

Reply:
Parameter makes more sense, as that reference value is not part of the builder (and it wouldn't make sense if someone used the builder for their own thing).

Reply:
Yes, I have added this as a param, it seems to always be 200 in vanilla, but I can see it being useful for mods to use.